src/libraries/System.ComponentModel.TypeConverter/tests/TypeDescriptorTests.cs

```diff
@@ -1141,6 +1141,30 @@ public void DerivedPropertyAttribute()
             Assert.Equal("Derived", descriptionAttribute.Description);
         }
 
+        [Fact]
+        public void PropertyFilterAttributeMatch()
+        {
+            Assert.Equal(3, TypeDescriptor.GetProperties(typeof(POCO), new[] { new PropertyFilterAttribute() }).Count);
+        }
+
+        public sealed class PropertyFilterAttribute : Attribute
+        {
+            public override bool Equals(object value) => ReferenceEquals(this, value);
+
+            public override int GetHashCode() => throw new NotImplementedException();
+
+            public override bool Match(object value) => true;
+
+            public static readonly PropertyFilterAttribute Default = new PropertyFilterAttribute();
+        }
+
+        public class POCO
```

Comment:
Can this class name be more descriptive? Maybe PropertyFilterAttributeMatchPoco?